### PR TITLE
feat: JSR + `/x` support

### PR DESCRIPTION
**Pull request by iuioiua**
The script was used for the publishing of the first version. Check it out: https://jsr.io/@bartlomieju/postgres@0.17.2

### CODE REVIEW

File: deno.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "lock": false,
+  "name": "@bartlomieju/postgres",
```
Comment (bombillazo) [inline deno.json:3]: Shouldnt this be `denodrivers/postgres`?

**Review by bombillazo — COMMENTED** [review]
LGTM, will this also take care of replacing the comment examples?

File: deno.json (hunk 1 above)
Comment (iuioiua) [reply deno.json:3]: @bartlomieju wanted it under his own scope.

**Comment by iuioiua** [comment]
> LGTM, will this also take care of replacing the comment examples?

Yep!

**Review by bombillazo — APPROVED** [review]

**Comment by iuioiua** [comment]
Bartek, I'm going to merge now. Please feel free to review it afterwards.

File: .github/workflows/publish_jsr.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+name: Publish to JSR
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  DENO_UNSTABLE_WORKSPACES: true
```
Comment (bartlomieju) [inline .github/workflows/publish_jsr.yml:9]: This is not needed anymore

**Review by bartlomieju — COMMENTED** [review]
Looks good, thanks!

**Pull request merged**